src/test/kotlin/net/maizegenetics/phgv2/cli/CreateFastaFromHvcfTest.kt

```diff
@@ -349,28 +350,110 @@ class CreateFastaFromHvcfTest {
 
         val dbPath = "${TestExtension.testOutputFastaDir}/dbPath"
 
-        createFastaFromHvcf.buildFastaFromHVCF(dbPath, "${TestExtension.testOutputFastaDir}/Ref_Test_output.fa", "composite","",refHVCFFileName,"")
+        createFastaFromHvcf.buildFastaFromHVCF(dbPath, "${TestExtension.testOutputFastaDir}", "composite","",refHVCFFileName,"")
 
         //Compare the composite against the truth input
         val truthFasta = NucSeqIO("data/test/smallseq/Ref.fa").readAll()
-        val outputFastaComposite = NucSeqIO("${TestExtension.testOutputFastaDir}/Ref_Test_output.fa").readAll()
+        val outputFastaComposite = NucSeqIO("${TestExtension.testOutputFastaDir}/Ref_composite.fa").readAll()
         for(chr in truthFasta.keys) {
             assertEquals(getChecksumForString(truthFasta[chr]!!.seq()), getChecksumForString(outputFastaComposite[chr]!!.seq()))
         }
 
         //build a haplotype fasta as well
-        createFastaFromHvcf.buildFastaFromHVCF(dbPath, "${TestExtension.testOutputFastaDir}/Ref_Test_output_hap.fa", "haplotype","",refHVCFFileName,"")
+        createFastaFromHvcf.buildFastaFromHVCF(dbPath, "${TestExtension.testOutputFastaDir}", "haplotype","",refHVCFFileName,"")
 
         //get truth hashes:
         val truthHashes = altHeaders.values.map { it.id }.toSet()
         //load in the haplotypes and build hashes
-        val outputFastaHaplotypes = NucSeqIO("${TestExtension.testOutputFastaDir}/Ref_Test_output_hap.fa").readAll()
+        val outputFastaHaplotypes = NucSeqIO("${TestExtension.testOutputFastaDir}/Ref_haplotype.fa").readAll()
+        outputFastaHaplotypes.values.map { getChecksumForString(it.seq()) }.toSet().forEach{
+            assertTrue(truthHashes.contains(it))
+        }
+
+    }
+
+    @Test
+    fun testBuildFastaFromHVCF_dirInput() {
```

Comment:
You should use kotlin naming conventions here with camelCase not _.  Same for the single file one as well.

Reply:
The name is camel-cased except at the end.  I used the underscore to make it clear that one case is testing input from  a directory of files and the other is just a single file.  I prefer to keep the names as they are currently as it is much easier to identify the 2 different test cases.